prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -129,6 +130,11 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
     this.toastService.openSuccessToast('Your GPID has been copied to clipboard');
   }
 
+  public hasErrors() {
+    return Object.values(this.getEnrolmentErrors(this.enrolment))
+      .some((value) => value);
```

Comment:
Should call it what it is: `.some(error => error)`

Reply:
Renamed, also I am only picking two of the errors now as checking all of them introduced a bug